apis/certificatemanager/v1beta1/certificatemanagerdnsauthorization_types.go

```diff
@@ -90,7 +92,7 @@ type CertificateManagerDNSAuthorizationObservedState struct {
 // +kubebuilder:printcolumn:name="Ready",JSONPath=".status.conditions[?(@.type=='Ready')].status",type="string",description="When 'True', the most recent reconcile of the resource succeeded"
 // +kubebuilder:printcolumn:name="Status",JSONPath=".status.conditions[?(@.type=='Ready')].reason",type="string",description="The reason for the value in 'Ready'"
 // +kubebuilder:printcolumn:name="Status Age",JSONPath=".status.conditions[?(@.type=='Ready')].lastTransitionTime",type="date",description="The last transition time for the value in 'Status'"
-// +kubebuilder:metadata:labels="cnrm.cloud.google.com/managed-by-kcc=true";"cnrm.cloud.google.com/stability-level=stable";"cnrm.cloud.google.com/tf2crd=true"
```

Comment:
have you run dev/tasks/generate-crds or did you add this resource inthe excluded_resources list?

Reply:
I ran `dev/tasks/generate-crds` already, it is not in the included list.

Reply:
the tag was in alpha api also. I remove it, and regen crds